src/testdir/test_vim9_class.vim

```diff
@@ -955,6 +955,28 @@ def Test_class_new_with_object_member()
   END
   v9.CheckSourceFailure(lines, 'E1013:')
 
+  lines =<< trim END
+      vim9script
+
+      class C
+        this.str: string
+        def MethodA(this.str)
+        enddef
+      endclass
+
+      def Check()
+        try
+          var c = C.new()
+          c.MethodA('abc')
+        catch
+          assert_report($'Unexpected exception was caught: {v:exception}')
+        endtry
+      enddef
+
+      Check()
+  END
+  v9.CheckSourceFailure(lines, 'E1390:')
```

Comment:
I started using complete error messages for verification as this makes the test easier to read and understand.  Can you include the complete the error message here?